.github/workflows/coverage.yml

```diff
@@ -9,7 +9,51 @@ on:
   pull_request:
 
 jobs:
-  call_code_coverage:
+  combined_coverage:
     uses: yext/slapshot-reusable-workflows/.github/workflows/coverage.yml@v1
+    with:
+      test_script: npm run test
     secrets:
       caller_github_token: ${{ secrets.GITHUB_TOKEN }}
+  individual_coverage_percentages:
+    runs-on: ubuntu-latest
+    needs: combined_coverage
+    steps:
+      - uses: actions/checkout@v3
+      - name: Download the current coverage
+        uses: actions/download-artifact@v3
+        with:
+          name: coverage
+          path: coverage/
+      - name: Get Visual Coverage Percentage
+        uses: yext/slapshot-reusable-workflows/get-coverage-percent@v1
+        with:
+          coverage-file: coverage/visual/lcov.info
+        id: parse-visual-coverage
+      - name: Get Unit Coverage Percentage
+        uses: yext/slapshot-reusable-workflows/get-coverage-percent@v1
+        with:
+          coverage-file: coverage/unit/lcov.info
+        id: parse-unit-coverage
+      - name: Get Combined Coverage Percentage
+        uses: yext/slapshot-reusable-workflows/get-coverage-percent@v1
+        with:
+          coverage-file: coverage/lcov.info
+        id: parse-combined-coverage
+      - name: Find Comment
+        uses: peter-evans/find-comment@v2
+        id: fc
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: 'github-actions[bot]'
+          body-includes: Current visual coverage
```

Comment:
which comment is this finding? do we initially make one with just the visual coverage?

Reply:
this is replacing the previous comment so there would only ever be 1 comment from github action regarding code coverage percentage.